src/app/proposals/draft/[id]/page.tsx

```diff
@@ -28,6 +29,7 @@ export default async function DraftProposalPage({
   }
 
   const draftProposal = await fetchDraftProposal(parseInt(params.id));
+  const proposalTypes = await fetchProposalTypes();
```

Comment:
@mcgingras I was getting a TypeScript error here. It was complaining that `DraftFormServer.tsx` was not returning a valid JSX in the suspense state. Anyway, this is a quick and dirty workaround. I am also now realizing that we don't even need the `DraftFormServer.tsx` wrapper.

Reply:
Oh interesting. I was getting that error too sometimes but it would still build, I was chalking it up to my tsconfig possibly not being up to date with server components. Yea, maybe we don't need it, I think the intention was having a wrapper to allow us to fetch proposal types while the rest of the real component could be a client component